src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -1296,15 +1192,44 @@ public static CultureInfo GetCultureInfo(string name, string altName)
                 throw new ArgumentNullException(nameof(altName));
             }
 
-            CultureInfo? retval = GetCultureInfoHelper(-1, name, altName);
-            if (retval == null)
+            name = CultureData.AnsiToLower(name);
+            altName = CultureData.AnsiToLower(altName);
+            string nameAndAltName = name + "\xfffd" + altName;
+            Dictionary<string, CultureInfo> nameTable = CachedCulturesByName;
+            CultureInfo? result;
+
+            lock (nameTable)
+            {
+                if (nameTable.TryGetValue(nameAndAltName, out result))
+                {
+                    return result;
+                }
+            }
+
+            try
```

Comment:
>try [](start = 12, length = 3)

same as previous comment.

Reply:
How would CreateCultureNoThrow work here?  I don't see an overload that takes name and altName.

Reply:
Oh, I think you were referring to the other comment about adding such an overload, and it not being worth it.  Agreed.